content/kubermatic/master/tutorials_howtos/upgrading/upgrade_from_2.18_to_2.19/_index.en.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Upgrading from 2.18 to 2.19"
+date = 2021-01-07T08:00:39+02:00
+weight = 105
++++
+
+## Helm chart changes
+
+With KKP release 2.19, we're moving to use upstream Helm charts for some of the components. Following list describes the changes to the components and required actions to perform an upgrade.
+
+Note: the upstream charts will be downloaded during the deployment process when using the installer. If you're installing manually, the dependencies have to be downloaded separately for each of the charts mentioned below using command `helm dependency build <chart_location>`
+
+### cert-manager
+
+Cert Manager is now using upstream Helm chart version 1.5.2 - [documentation](https://cert-manager.io/docs/).
```

Comment:
It's always called `cert-manager`, never `Cert Manager`: https://cert-manager.io/docs/faq/style/